### PR TITLE
[DOC] RDoc for process.c

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

**Review by peterzhu2118 — APPROVED** [review]

File: process.c [hunk 1]
```diff
@@ -8639,38 +8639,108 @@ get_PROCESS_ID(ID _x, VALUE *_y)
...
+ *  If +signal+ is:
+ *
+ *  - A non-negative integer, or a signal name or symbol
+ *    without prefixed <tt>'-'</tt>,
+ *    each process with process ID +id+ is signalled.
+ *  - A negative integer, or a signal name or symbol
+ *    with prefixed <tt>'-'</tt>,
+ *    each process group with group ID +id+ is signalled.
+ *
+ *  The signal non-negative integers and strings are:
```
Comment (nobu) [inline process.c:8685]: Supported signals are platform dependent, I don't think list them up here.
Also the actual values too of course.

File: process.c [hunk 2]
```diff
@@ -8639,38 +8639,108 @@ get_PROCESS_ID(ID _x, VALUE *_y)
...
+ *
+ *  - +0+ (<tt>'SIGEXIT'</tt>): Exit the current process.
```
Comment (nobu) [inline process.c:8687]: This is not a real signal, nor exit the process.
- For `Process.kill`, `0` examines if the destination process is alive or not, but `SIGEXIT` is nog recognized as a signal.
- For `Signal.trap`, Hook for when the current process exits.

File: process.c (hunk 1 above)
Comment (BurdetteLamar) [reply process.c:8685]: Will look into how best to make this change.  (Can't reopen a PR, right?)

File: process.c (hunk 2 above)
Comment (BurdetteLamar) [reply process.c:8687]: Ditto,

**Pull request merged**